Polyhedron/demo/Polyhedron/Scene_polylines_item.cpp

```diff
@@ -367,7 +376,8 @@ Scene_polylines_item::supportsRenderingMode(RenderingMode m) const {
 // Shaded OpenGL drawing: only draw spheres
 void
 Scene_polylines_item::draw(CGAL::Three::Viewer_interface* viewer) const {
-
+  if(!visible())
+    return;
     if(!are_buffers_filled)
```

Comment:
There is a minor warning, because of the indentation:
```
.../platforms/ArchLinux-CXX14/test/Polyhedron_Demo/Scene_polylines_item.cpp: In member function 'virtual void Scene_polylines_item::draw(CGAL::Three::Viewer_interface*) const':
.../platforms/ArchLinux-CXX14/test/Polyhedron_Demo/Scene_polylines_item.cpp:379:3: warning: this 'if' clause does not guard... [-Wmisleading-indentation]
...
...
.../platforms/ArchLinux-CXX14/test/Polyhedron_Demo/Scene_polylines_item.cpp:381:5: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the 'if'
     if(!are_buffers_filled)
     ^~
```